lib/rules/declaration-property-value-no-unknown/index.js

```diff
@@ -108,7 +109,11 @@ const rule = (primary, secondaryOptions) => {
 				return;
 			}
 
-			const { error } = forkedLexer.matchProperty(prop, cssTreeValueNode);
+			const { error } =
+				decl?.parent?.type === 'atrule'
+					? // @ts-expect-error -- CSSTree types are not complete
+					  forkedLexer.matchAtruleDescriptor(decl.parent.name, prop, cssTreeValueNode)
```

Comment:
I'll send a patch to `@types/css-tree`. 💪🏼

Reply:
Thanks! Note that more functions from the `Lexer` class are missing, so maybe you can add them all in one batch?

Also, I used `@ts-expect-error` instead of `@ts-ignore`, so that this PR can be merged first and will throw an error once the `css-tree` types are fixed.

Reply:
[`checkAtrulePrelude`](https://github.com/csstree/csstree/blob/612cc5f2922b2304869497d165a0cc65257f7a8b/lib/lexer/Lexer.js#L244) might be needed for #6560.

Reply:
I've opened PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64410. Any feedback is welcome if you have.